ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -122,6 +122,19 @@ tests:
     test:
     - ref: openshift-extended-test
     - ref: rosa-terraform-e2e-tests-upload-polarion
+- as: rosa-sts-ad-critical-high-f3
+  cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CASE_LABEL_FILTER: (Critical,High)&&(day1-post,day2)&&!Exclude
```

Comment:
Why run only Critical and High?

Reply:
@amalykhi We will have another job to run (Medium,Low) which won't run frequently because they are not important. You can create a ticket to track the job creation